src/express-recaptcha.js

```diff
@@ -0,0 +1,90 @@
+import Https from 'https'
+export default class Recaptcha{
```

Comment:
I know it's tempting but `default` doesn't play well with `require`, it it's somewhat annoying to use.

```const Recaptcha = require('recaptcha').default;```

Our team generally prefers to just `export`, plays nice with destructuring too!

```const { Recaptcha } = require('recaptcha')```